screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
```

Comment:
You didn't properly create this file. You need to add the `.js` extension. Right now GitHub doesn't know that it's a Javascript file.

Also change the name of the file to `ShoppingList.js`